exporter/datadogexporter/traces_exporter.go

```diff
@@ -212,6 +220,12 @@ func newTraceAgentConfig(ctx context.Context, params exporter.Settings, cfg *Con
 	acfg.PeerTagsAggregation = cfg.Traces.PeerTagsAggregation
 	acfg.PeerTags = cfg.Traces.PeerTags
 	acfg.MaxSenderRetries = 4
+	if traceCustomHTTPFeatureGate.IsEnabled() {
+		params.Logger.Warn("Experimental feature: datadog exporter trace export uses a custom HTTP client from the exporter HTTP configs")
```

Comment:
This warning is not actionable, some people want to have a way to remove warnings if they are not actionable. Would it be fine to make it info level?
```suggestion
		params.Logger.Info("Experimental feature: datadog exporter trace export uses a custom HTTP client from the exporter HTTP configs")
```